app/controllers/constituencies_controller.rb

```diff
@@ -38,6 +38,8 @@ def show
 
     @party = @current_party ? @current_party : @party.first
 
+    @region = @constituency.region&.first
```

Comment:
is first the correct way to get this? If it's in more than one region is there a better way?

Reply:
Would it be better to display multiple regions? OR at least have the logic in the front-end and backend to do it if there are multiple regions?